modules/ROOT/pages/policy-openid-connect.adoc

```diff
@@ -8,98 +8,182 @@ endif::[]
+
+You can enable the OpenID Connect OAuth 2.0 Token Enforcement policy to use the gateway proxy settings by specify the following property when starting the Mule runtime engine: 
+
+`anypoint.platform.external_authentication_provider_enable_proxy_settings=<true|false(default)>`
+
+When you enable this property, the policy uses the Mule proxy settings, if you have specified the following parameters:
+
+* `anypoint.platform.proxy_host=localhost`
+* `anypoint.platform.proxy_port=8080`
+
+== Configuring Policy Parameters
+
+When you apply the policy to your API from the UI, a list of parameters are displayed based on whether your environment includes Mule or xref:service-mesh::index.adoc[Non-Mule application managed by Anypoint Service Mesh].
+
+=== Configuring the Parameters for Mule Applications
+
+For Mule applications, the following parameters are displayed based on whether your environment includes Mule applications:
+
+[%header%autowidth.spread,cols="30,40,30"]
+|===
+| Element | Description | Example
+| Scopes | A space-separated list of supported scopes. | READ, WRITE, READ and WRITE 
+| Scope Validation Criteria | Determines if the token must contain all defined scopes or just one.
+Values: `Contains all scopes` and `Contains any scope` | 
+In this example, the value `READ,WRITE` was selected for the scope element.
+If the ‘Contains all scopes’ value is selected, the tokens with the scopes `READ` and `WRITE` are accepted. If the ‘Contains any scopes’ value is selected, the tokens with the scope `READ, WRITE, and READ, WRITE` are accepted. 
+| Validate TLS Certificate | Enables third-party authentication server TLS validation. | If selected, the communication with the token validation endpoint uses the TLS protocol.
+| Expose Headers | In a proxy scenario, defines if headers should be exposed in the request to the backend. User properties returned by the federation server are sent as headers to the backend with the 'X-AGW-' prefix. | See the <<leveraging-TVE-information,Leveraging Token Validation Endpoint Information for Mule Applications>> for more information about this field.
+| Skip Client Id Validation | Skips client application's API contract validation. | If selected, the step 4 of the diagram in the <<how-this-policy-works,How This Policy Works>> section is skipped.
+|===
+
+=== Configuring the Parameters for Non-Mule Applications (Anypoint Service Mesh)
+
+For (non-Mule applications that are managed by Anypoint Service Mesh, you configure the OpenID Connect OAuth 2.0 Token Enforcement policy in the same way, excluding the following differences:
+
+[%header%autowidth.spread,cols="30,40,30"]
+|===
+| Element | Description | Example
+| Scopes | A space-separated list of supported scopes. | READ, WRITE, READ and WRITE
+| Skip Client Id Validation | Skips client application's API contract validation. | If selected, the step 4 of the diagram in the <<how-this-policy-works,How This Policy Works>> section is skipped.
+|===
+
+
+== FAQs
```

Comment:
I think we are trying to avoid FAQs and incorporate the info within them in the appropriate place in the running narrative.